frontend/src/components/HospitalInventoryEdit.tsx

```diff
@@ -0,0 +1,194 @@
+import React, { useEffect, useState } from 'react'
+import Col from 'react-bootstrap/Col'
+import Form from 'react-bootstrap/Form'
+import Card from 'react-bootstrap/Card'
+import Button from 'react-bootstrap/Button'
+import { useParams } from 'react-router-dom'
+import style from './HospitalInventoryEdit.module.css'
+import { HospitalInventory, HospitalAddress, fetchHospitalInventory, fetchHospitalAddress } from '../apis/infomed'
+
+
+interface HospitalInventoryParams {
+  hospitalId: string
+}
+
+export const HospitalInventoryEdit = ({hospitalId}: HospitalInventoryParams) => {
```

Comment:
Nitpick, optional: Can we rename it to `HospitalInventoryEditor` along with the entire file?